### ISSUE TITLE
new leptonica.net doesn't work in tesseract.net

### ISSUE THREAD

**Issue by yasindn** [tvn-cosine/tesseract.net#32]
Hi
leptonica.net updated by @fdncred. added some methods and new binary file "leptonica-1.77.0.dll"
I want to use last version of leptonica.net in tesseract.net, but I got error "unable to load dll"

**Comment by yasindn** [comment]
Yes, it can't. but all dll files available in the root directory.
I guess I found the problem.
the last version of [leptonica ](https://cppan.org/pvt.cppan.demo.danbloomberg.leptonica)is 1.76.0 and depend on 6 other packages. How do you create "leptonica-1.77.0.dll"?
pvt.cppan.demo.google.tesseract.libtesseract.dll  have dependencies to pvt.cppan.demo.danbloomberg.leptonica. and pvt.cppan.demo.danbloomberg.leptonica.dll depend on some other packages. 
Can you explain how to create or create new libtesseract dll that depend on "leptonica-1.77.0.dll."

**Comment by fdncred** [comment]
It's compiled statically so you don't have to have all those other cppan files.

**Comment by yasindn** [comment]
When I include all cppan leptonica and dependencies I didn't get the error. but when I am using leptonica1.77.dll and cppan tesseract I got the error.

I don't have. but I am working on master repo offline.
I compiled leptonica.net, get 2 files. leptonica.dll and leptonica1.77.dll
Next, I removed the leptonica NuGet package from tesseract.net and add leptonica.dll 
Finally, I am using leptonica.dll and leptonica1.77.dll and cppan.tesseract on tesseract consol

**Comment by fdncred** [comment]
It sounds like tesseract.net needs to point to leptonica1.77 instead of 1.76 and all the cppan files.
